IgOmeProfiling_pipeline.py

```diff
@@ -265,7 +265,7 @@ def run_pipeline(fastq_path, barcode2samplename_path, samplename2biologicalcondi
     parser.add_argument('--prefix_length_in_clstr', default=20, type=int,
                         help='How long should be the prefix that is taken from the clstr file (cd-hit max prefix is 20)')
     parser.add_argument('--multi_exp_config_inference', type=str, help='Configuration file for inference motifs phase to run multi expirements')
-    parser.add_argument('--precent_random_hit_pssm', type=float, default=0.05, help='Precent of random hits per PSSM, for calculate cutoffs')
+    parser.add_argument('--cutoff_random_peptitdes_percentile', type=float, default=0.05, help='Set the top percentile of random peptides and define it a cutoff score')

```

Comment:
Change help to "Calculate cutoff (hit threshold) from random peptides' top percentile score"